src/services/GooglePlacesService.ts

```diff
@@ -1,183 +1,48 @@
+import { Place, PlacePhoto } from '@googlemaps/google-maps-services-js';
 import axios, { AxiosError, AxiosResponse } from 'axios';
 
-import { RestaurantDetails } from '../types/restaurant';
-import { RestaurantInfo } from '../types/RestaurantInfo';
+import Routes from '../utils/routes';
 
-/* eslint-disable  @typescript-eslint/no-explicit-any */
 class GooglePlacesService {
-  // maintains a mapping of restaurant placeIDs with associated restaurant details (see types)
-  private currRestaurants: Map<string, RestaurantDetails>;
```

Comment:
Now this class is much simpler, leaving the responsibility of handling NOT_FOUND's to the BE, using interfaces etc